src/ui/components/ember-cli-diff/component.ts

```diff
@@ -35,16 +33,28 @@ export default class EmberCliDiff extends Component {
   }
 
   didInsertElement() {
-    $("select").select2({
-      data: this.createHashForSelect2(this.json)
-    });
+    this.loadTags().then((tags) => {
+      this.json = tags;
 
-    $("#from").val(this.from).trigger("change")
-    $("#to").val(this.to).trigger("change")
+      $("select").select2({
+        data: this.createHashForSelect2(tags)
+      });
 
-    if (this.from && this.to) {
-      this.loadDiff();
-    }
+      $("#from").val(this.from).trigger("change")
+      $("#to").val(this.to).trigger("change")
+
+      if (this.from && this.to) {
+        this.loadDiff();
+      }
+    });
+  }
+
+  loadTags() {
+    return fetch(`https://api.github.com/repos/ember-cli/ember-new-output/tags?per_page=100`, {
```

Comment:
Right now there is a 60 requests to github per hour limit on the implementation. This change will duplicate usage. Anyway, it might be worth giving it try and if it's a problem, we can improve how we query github.